docs/components/links/README.md

```diff
@@ -298,6 +301,7 @@ Use link labels that describe the link’s destination when clicked or tapped:
   - For example, if the link text is “Learn more”, then the `alt` text could be “Learn more about pick up in store”
 - Don’t capitalize links. Some screen readers read capitalized text letter-by-letter. Instead, use sentence case
 - Restrict the number of text links on a page. Screen reader will read all the links on a page
+- Be sure the copy used in link buttons clearly communicates the action they'll perform
```

Comment:
we can probably just update line 298 to say smth like `Use descriptive text for links, so that the user knows where they are going or what action they are taking`

Reply:
Sounds good to me